<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowDialog</class>
 <widget class="QDialog" name="ShowDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Описание задачи</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="indexLabel">
     <property name="text">
      <string>Индекс</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="addTagPushButton">
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="userLabel">
     <property name="text">
      <string>Пользователи</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="tagLabel">
     <property name="text">
      <string>Тэги</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>Заголовок</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="titleLineEdit"/>
   </item>
   <item row="5" column="1" rowspan="2">
    <widget class="QTextEdit" name="userTextEdit"/>
   </item>
   <item row="7" column="0" colspan="3">
    <widget class="QTextEdit" name="showTextEdit"/>
   </item>
   <item row="8" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="showButtonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1" rowspan="2">
    <widget class="QTextEdit" name="tagTextEdit"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="calendarLabel">
     <property name="text">
      <string>Дедлайн</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="addUserPushButton">
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="indexLineEdit"/>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="deleteTagPushButton">
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="deleteUserPushButton">
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QCalendarWidget" name="calendarWidget"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>showButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>ShowDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>ShowDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
